packages/paste-core/components/disclosure/src/index.tsx

```diff
@@ -1,3 +1,5 @@
 export * from './Disclosure';
 export * from './DisclosureHeading';
 export * from './DisclosureContent';
+export * from './DisclosureContext';
```

Comment:
Why do we need to expose this to the consumer?

Reply:
We don't. I'll remove.